pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -120,21 +120,24 @@ def update_predictoors_table_on_search(
         ]
 
         if "show-favourite-addresses.value" in dash.callback_context.triggered_prop_ids:
-            custom_predictoors = formatted_predictoors_data.filter(
-                formatted_predictoors_data["full_addr"].is_in(
-                    app.data.favourite_addresses
+            if len(stored_predictoor_addrs) > 0:
```

Comment:
I would move this to a helper function that retrieves custom_predictoors either from the cache or ppss

Reply:
It would be hard to move something that is using the cache separately, because the cache can only be accessed inside the callback